### ISSUE TITLE
Validation missing: runtime environments and runtime libraries should only contain .n4jsd files

### ISSUE THREAD

**Issue by NumberFourAG** [NumberFour/n4js#104]
IDEBUG-827
 I had already implemented such a validation, but our tests make heavy use of .n4js (not .n4jsd) files in runtime libraries (almost 1000 test failures). Tests need to be adjusted. Hence this separate task.

**Comment by mor-n4** [comment]
once this is done, check if #30 and #50 can be closed as well.

**Comment by dbo** [comment]
The discussed changes have landed.
